### PR TITLE
[editor-common, common, gallery] fix: link data values

### PR DESCRIPTION
**Pull request by maimonav**
1. Gallery modal fix using `uiSettings`.
2. Link values save fix.

### CODE REVIEW

File: packages/common/web/src/test/linkConverters.spec.ts [hunk 1]
```diff
@@ -29,9 +29,9 @@ describe('Test link converters', () => {
     expect(converted).toEqual(relObject);
   });
 
-  it('should convert target undefined to undefined', () => {
+  it('should convert target undefined to true', () => {
```
Comment (shaulgo) [inline packages/common/web/src/test/linkConverters.spec.ts:32]: This is very unintuitive with the name of the function.

**Review by shaulgo — CHANGES_REQUESTED** [review]

File: packages/plugin-gallery/web/src/components/gallery-controls/gallery-image-settings.js [hunk 2]
```diff
@@ -74,12 +74,16 @@ class ImageSettings extends Component {
       visibleRightArrow,
       uiSettings,
```
[the remark this answers is not in the document]
Comment (maimonav) [reply packages/plugin-gallery/web/src/components/gallery-controls/gallery-image-settings.js:75]: I'm not sure if it always was that way, but it has been like that for a while.

File: packages/common/web/src/test/linkConverters.spec.ts (hunk 1 above)
Comment (maimonav) [reply packages/common/web/src/test/linkConverters.spec.ts:32]: This is the default result, if target isn't defined then by default it's true.

Comment (shaulgo) [reply packages/common/web/src/test/linkConverters.spec.ts:32]: I don't love it. But this should will be removed when we migrate completely to new schema?

Comment (maimonav) [reply packages/common/web/src/test/linkConverters.spec.ts:32]: No, this is needed for the link panels.

**Review by shaulgo — APPROVED** [review]

**Pull request merged**